### PR TITLE
A test for alias + extend in schemas

### PR DESCRIPTION
**Pull request by piotrswigon**
Discussed this use case with Daniel and wanted to ensure it actually works :)

### CODE REVIEW

File: src/runtime/tests/schema-test.ts [hunk 1]
```diff
@@ -444,6 +444,21 @@ describe('schema', () => {
...
+        Text name
+        
+      schema Accident extends EventAlias
+        Number financialCost
+      
+      schema Crisis extends Accident`);
+    
+    assert.deepEqual(manifest.findSchemaByName('EventAlias').names, ['Event', 'Occurrence']);
+    assert.deepEqual(manifest.findSchemaByName('Accident').names, ['Accident', 'Event', 'Occurrence']);
+    assert.deepEqual(manifest.findSchemaByName('Crisis').names, ['Crisis', 'Accident', 'Event', 'Occurrence']);
```
Comment (mmandlis) [inline src/runtime/tests/schema-test.ts]: can you, please, also add an assert to test the fields? thanks.

**Comment by Cypher1** [comment]
Seconding @mmandlis 's request, otherwise LGTM

File: src/runtime/tests/schema-test.ts (hunk 1 above)
Comment (piotrswigon) [reply src/runtime/tests/schema-test.ts]: Done :)

**Comment by piotrswigon** [comment]
PTAL again :)

**Comment by piotrswigon** [comment]
Can I have approval? :)

**Pull request merged**